CHANGELOG.md

```diff
@@ -2,14 +2,32 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased
+# Unreleased
 
-### Sprint 24 end 2015-06-01
+## Sprint 25 end 2015-06-22
 
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+25+Ending+June+22%2C+2015%22+label%3Aenhancement)
 
-#### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+24+Ending+June+1%2C+2015%22+label%3Aenhancement)
+- Container Integration: Kubernetes Events
+- Amazon: Added M4 and t2.large instance types
+- UI: Added donut charts
+- Extracting C code to external gems
+  - MiqLargeFileLinux => [large\_file\_linux gem]( https://github.com/ManageIQ/large_file_linux)
+  - MiqBlockDevOps => [linux\_block\_device gems](https://github.com/ManageIQ/linux_block_device)
```

Comment:
I think this refactoring should go under Changed section.  We didn't really add it, we just moved it.